my.java

```diff
@@ -0,0 +1,20 @@
+interface my
```

Comment:
First letter of interface /class name should be capital

Reply:
ohk